processor/stream/approved-es-documents/testV2IntakeIntegrationOptionalTimestamps.approved.json

```diff
@@ -0,0 +1,146 @@
+{
+    "events": [
+        {
+            "@timestamp": "2018-08-01T10:00:00Z",
```

Comment:
I assume this should test that a given timestamp is not overwritten, but if it is missing the approrpiate timestamp is set. Please use two different timestamps for this (atm the default timestamp is the same as the one used in the testdata file).